Small fix in orchestrating-third-party-services-with-oauth2.adoc (#127)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc

. When all the services are running, use the following `curl` commands to run the `currency-exchange-workflow`:
+
--

> **Example 1)** Successful execution when calculating the currency exchange from `EUR` to `USD`

Reviewer: ```suggestion
The following is an example of successful execution when calculating the currency exchange from `EUR` to `USD`:
```

----
====

> **Example 2)** Unsupported currency error when calculating the currency exchange from `EUR` to `MXN`

Reviewer: ```suggestion
The following is an example of an unsupported currency error when calculating the currency exchange from `EUR` to `MXN`:
```

In the previous examples, the currencies supported by the `currency-exchange-workflow` include `EUR`, `USD`, `JPY`, `GBP`, `CAD`, `BRL`, and `AUD`. However, the `acme-financial-service` REST service can resolve any type of currency exchange. This is an example of a Serverless Workflow implementing the intermediate data filtering, transforming, and validations.
====

The following is an example of the occurrence of unexpected errors when accessing `acme-financial-service`:

> To execute, you must go to the terminal window where you started the `acme-financial-service` and stop the service using `CTRL+C`.
> 

Reviewer 2: ```suggestion
Before you launch the command, you must go to the terminal window where you started the `acme-financial-service` and stop the service using `CTRL+C`.

```

In the previous examples, the currencies supported by the `currency-exchange-workflow` include `EUR`, `USD`, `JPY`, `GBP`, `CAD`, `BRL`, and `AUD`. However, the `acme-financial-service` REST service can resolve any type of currency exchange. This is an example of a Serverless Workflow implementing the intermediate data filtering, transforming, and validations.
====

**Example 3)** Unexpected errors when accessing `acme-financial-service`

> To execute this case, first you must go the terminal window where you started the `acme-financial-service` and stop it with CTRL+C.

Reviewer: ```suggestion
To execute the previous case, you must go to the terminal window where you started the `acme-financial-service` and stop the service using `CTRL+C`.
```
